specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json

```diff
@@ -226,9 +226,65 @@
         "daily",
         "hourly",
         "minutely",
-        "secondly"
+        "secondly",
+        "microsecond"
       ]
     },
+    "ImputeMode": {
+      "type": "string",
+      "description": "can be one of auto, previous, linear, fixed, zero or notFill.",
+      "x-nullable": false,
+      "x-ms-enum": {
+        "name": "TimeGranularity",
```

Comment:
There's already an enum type with this name, one of them will have to be renamed.

Reply:
Updated.